ouroboros-consensus/src/Ouroboros/Consensus/HardFork/History/EpochInfo.hs

```diff
@@ -16,52 +16,31 @@ import           GHC.Stack
 
 import           Cardano.Slotting.EpochInfo.API
 
-import           Ouroboros.Consensus.Util.IOLike
-
-import           Ouroboros.Consensus.HardFork.History.Caching
 import           Ouroboros.Consensus.HardFork.History.Qry
 import           Ouroboros.Consensus.HardFork.History.Summary
 
 {-------------------------------------------------------------------------------
   Translation to EpochInfo
 -------------------------------------------------------------------------------}
 
--- | Construct 'EpochInfo' from a function that returns the hard fork summary
---
--- When a particular request fails with a 'PastHorizon' error, we ask for an
--- updated summary, in the hope that the ledger state has advanced. If the query
--- /still/ fails with that updated summary, the error is thrown as an exception.
-summaryToEpochInfo :: forall m xs. (MonadSTM m, MonadThrow (STM m))
-                   => STM m (Summary xs) -> m (EpochInfo (STM m))
-summaryToEpochInfo =
```

Comment:
Huh. This was dead-code; only exported. I checked `master` on `cardano-node` and `cardano-db-sync` and found no uses. Those were the main clients I suspected had any chance of using it.